gatsby-theme-landing-page/README.md

```diff
@@ -38,9 +38,25 @@ CONTENTFUL_SPACE_ID="<SPACE_ID>"
 CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
 ```
 
-<!-- TODO: update these instructions once the JSON file is created -->
+Import the content model and demo data to your Contentful space
 
-Upload the `data/sample-data.json` to your Contentful space to set up the content model. Then, create your first landing page in Contentful.
+1. Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
+
+2. Add your space ID to `contentful/import-demo-config.json` and `contentful/import-model-only-config.json`
```

Comment:
I wonder if there's a way to leverage env vars for the contentful-cli (thinking about ways to reduce steps here) 🤔